velox/exec/Task.h

```diff
@@ -1109,6 +1109,9 @@ class Task : public std::enable_shared_from_this<Task> {
 
   // Indicates whether the spill directory has been created.
   std::atomic<bool> spillDirectoryCreated_{false};
+
+  // Stores unconsumed preloading splits to ensure they are closed promptly.
+  std::vector<std::shared_ptr<connector::ConnectorSplit>> preloadingSplits_;
```

Comment:
Let's use `folly::FastSet` here for simpler code

Reply:
Updated, thanks.